spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/procedures/BaseProcedure.java

```diff
@@ -144,6 +148,12 @@ protected SparkTable loadSparkTable(Identifier ident) {
     }
   }
 
+  protected Dataset<Row> loadDataSetFromTable(Identifier tableIdent, Map<String, String> options) {
+    String tableName = Spark3Util.quotedFullIdentifier(tableCatalog().name(), tableIdent);
+    // no need to validate the read options here since the reader will validate them
```

Comment:
I don't think we need this comment anymore since it is a pretty generic method now.